Makefile

```diff
@@ -97,13 +97,21 @@ docker-uploadproxy: docker
 docker-uploadserver: WHAT = cmd/cdi-uploadserver
 docker-uploadserver: docker
 
-docker-functest-image: docker-functest-image-http docker-functest-image-init
+docker-functest-image: docker-functest-image-http docker-functest-image-init docker-func-test-registry-populate docker-func-test-registry
 docker-functest-image-init: WHAT = tools/cdi-func-test-file-host-init
 docker-functest-image-init: docker
 docker-functest-image-http: WHAT = tools/cdi-func-test-file-host-http
 docker-functest-image-http: # no code to compile, just build image
 	./hack/build/build-cdi-func-test-file-host.sh && ./hack/build/build-docker.sh build ${WHAT}
 
+docker-func-test-registry-populate: WHAT = tools/cdi-func-test-registry-populate
```

Comment:
nit: all the other targets are using functest and not func-test so better change it to be the same